pvlib/solarposition.py

```diff
@@ -38,26 +38,33 @@ def get_solarposition(time, latitude, longitude,
     Parameters
     ----------
     time : pandas.DatetimeIndex
+
```

Comment:
@wholmgren did you really mean to add all of these extra newlines? Should I merge them, or was it a typo? I can't find the rationale in your comments, and it's inconsistent with docstrings everywhere else. Please let me know. Thanks!